src/main/java/uk/gov/companieshouse/pscdataapi/models/PscDeleteRequest.java

```diff
@@ -2,4 +2,50 @@
 
 public record PscDeleteRequest (String companyNumber, String notificationId, String contextId, String kind, String deltaAt) {
 
+    public static Builder builder() { return new Builder(); }
+
+    public static final class Builder {
+
+        private String companyNumber;
+        private String notificationId;
+        private String contextId;
+        private String kind;
+        private String deltaAt;
+
+        private Builder() {
+        }
+
+        public static Builder builder() {
```

Comment:
This shouldn't be needed because you have it declared on line 5

Reply:
done